components/CookieBanner.vue

```diff
@@ -21,11 +21,15 @@
 </template>
 
 <script setup>
+import { useGtm } from '@gtm-support/vue-gtm'
+
 const showBanner = useCookie('bg.showbanner', { default: () => true })
 const allowCookies = useCookie('bg.allowcookies', { default: () => false })
 function yesCookies() {
     showBanner.value = false
     allowCookies.value = true
+    const gtm = useGtm()
+    gtm.enable()
 }
 
 function noCookies() {
```

Comment:
should be `allowCookies.value = false` here?

Reply:
Great catch :D